### DISCUSSION TITLE
Reproject WRF data

### DISCUSSION THREAD

**Discussion by francobarrionuevoenv21** [pydata/xarray#9039] [Q&A]
Hi, guys! 

I'm trying to work with some WRF model data from the national weather service. After retrieving the dataset for an specific forecast from AWS, I get the following dataset

![image](https://github.com/pydata/xarray/assets/78272951/b06299e2-0354-4457-86e2-fc7a14f1f0da)

As you can see in metadata, it's projected in lambert conformal. The thing is that I need to reproject it into epsg 4326. I've tried using write_crs (from rioxarray) and reproject methods but with no luck. I don't understand if it's something I don't get about CRS in this dataset or just a dataset format issue. 

Any help is welcome. Thank you!!

**Comment by scottyhq** [comment]
Rioxarray (`ds.rio.reproject()`) is expecting a regular grid, but the WRF lat/lon spacing is irregular. So you'd first need to interpolate onto a regular grid - see relevant discussion https://github.com/corteva/rioxarray/discussions/426. The recommended solution in that discussion for WRF specifically was to use the more specialized `salem` library built on top of Xarray that has routines for working with WRF, and grid interpolation: https://salem.readthedocs.io/en/v0.3.10/wrf.html. Did you eventually find a solution @francobarrionuevoenv21 , if you're able to share the code you ended up using that would be great!
